rholang/src/main/bnfc/rholang_mercury.cf

```diff
@@ -125,12 +125,15 @@ GroundUri.     Ground ::= Uri ;
 token Uri ('`' ((char - ["\\`"]) | ('\\' ["`\\"]))* '`') ;
 -- Collections:
 CollectList.   Collection ::= "[" [Proc] Remainder "]" ;
-CollectTuple.  Collection ::= "(" [Proc] ")" ;
+CollectTuple.  Collection ::= Tuple;
 CollectSet.    Collection ::= "Set" "(" [Proc] ")" ;
 CollectMap.    Collection ::= "{" [KeyValuePair] "}" ;
 KeyValuePairImpl.  KeyValuePair ::= Proc ":" Proc ;
 separator KeyValuePair "," ;
 
+TupleSingle.    Tuple ::= "(" Proc ",)" ;
```

Comment:
This should be "," ")"

Reply:
I thought that `",)"` would be better because lexer would recognize it as single token while `"," ")"` could become ambiguous after `","` .

Reply:
There are no warnings (`shift/reduce`) during compilation.

Reply:
"," ")" won't cause a conflict because after the "," both cases need to shift in a new token, which will be ")" in the unit tuple case, and anything else in the non-unit tuple case.

The goal is to write a more forgiving grammar as long as it's not ambiguous. If there are conflicts with the split, I'll settle for the single token.

Reply:
:+1:  will do. Thanks.

Reply:
Changed the way you suggested but I am getting conflict:
```
Warning : *** Shift/Reduce conflict found in state #81
  between ListProc ::= (*) 
  and     Tuple ::= _SYMB_7 Proc _SYMB_20 (*) _SYMB_8 
  under symbol _SYMB_8
  Resolved in favor of shifting.
```